lib/systems/doubles.nix

```diff
@@ -12,11 +12,14 @@ let
     "mipsel-linux"
 
     "i686-cygwin" "i686-freebsd" "i686-linux" "i686-netbsd" "i686-openbsd"
+    "i586-cygwin" "i586-freebsd" "i586-linux" "i586-netbsd" "i586-openbsd"
+    "i486-cygwin" "i486-freebsd" "i486-linux" "i486-netbsd" "i486-openbsd"
+    "i386-cygwin" "i386-freebsd" "i386-linux" "i386-netbsd" "i386-openbsd"
```

Comment:
i386 doesn't work with latest glibc - better leave it off

Reply:
Should I purge all i386 support from the PR or just this i386 line?

Reply:
Just the i386-linux references, I think. Although I'm not sure whether other OSs actually still support i386. Very few computers still even exist that are i386 so it's largely a irrelevant 😄

Reply:
I've followed code to see what `doubles.nix` actually does, and... it does almost nothing.  Only `mesaPlatforms` appears to be used AFAICT.  Anyway I will revert all changes to this file and also `for-meta.nix` (which looks 100% dead).